tests/HotBackup/HotBackupCoordinatorTest.cpp

```diff
@@ -88,6 +88,21 @@ int countSubstring(std::string const& str, std::string const& sub) {
 std::vector<std::string> dbsPath{"arango", "Plan", "DBServers"};
 std::vector<std::string> colPath{"arango", "Plan", "Collections"};
 
+namespace {
+void checkAlphabetic(std::map<ServerID, ServerID> const& matches) {
```

Comment:
I think this could be replaced by `std::is_sorted()` (3) from https://en.cppreference.com/w/cpp/algorithm/is_sorted

Reply:
I will look into this this afternoon. The .first entries are sorted by definition, I need to check that also the .second entries are sorted. If this is possible with `is_sorted`, then I will use this.

Reply:
You can write a custom compare function, such as `std::is_sorted(container.begin(), container.end(), [](auto const& lhs, auto const& rhs) { return lhs.second < rhs.second; });`

Reply:
Yes, this works. Done it.